source/mainnet/net/resources/terms-and-conditions-dw.rst

```diff
@@ -4,18 +4,14 @@
 Terms and Conditions for Concordium Desktop Wallet
 ==================================================
 
-Effective from August 4, 2022
+Effective from September XX, 2022
```

Comment:
Is this right? Why september, when it hasn't even been released yet?

Reply:
Oh... this pull request is old.. I didn't notice 🙈

Reply:
Do we need to coordinate so this is updated in the software too?

Reply:
Good question. I don't know what we've done previously. I'll look into it sometime this week.

Reply:
@orhoj Now that you are back I would like to get this out of the backlog of PRs that I have. Mikael updated them ages ago. They need to be updated in DW and then DW and this PR released.